test/extensions/quic_listeners/quiche/platform/BUILD

```diff
@@ -23,7 +24,7 @@ envoy_cc_test(
 
 envoy_cc_test(
     name = "quic_platform_test",
-    srcs = ["quic_platform_test.cc"],
+    srcs = envoy_select_quiche(["quic_platform_test.cc"]),
```

Comment:
Can you confirm that if you make a typo to the Quiche implementation that `bazel build //source/exe/...` works? Also, CI is broken. Other than that, this looks like a good approach, thanks!

Reply:
Any idea why ci is broken?

bazel build //source/exe/... works on my machine with the diff below.  Actually, even bazel build //source/extensions/... worked, which is unexpected to me.

$ git diff
diff --git a/source/extensions/quic_listeners/quiche/platform/quic_string_impl.h b/source/extensions/quic_listeners/quiche/platform/quic_string_impl.h
index 9370a02b1..68bf14b4a 100644
--- a/source/extensions/quic_listeners/quiche/platform/quic_string_impl.h
+++ b/source/extensions/quic_listeners/quiche/platform/quic_string_impl.h
@@ -1,4 +1,4 @@
-#pragma once
+fdsfdsfdsfdsfds#pragma once
 
 #include <string>